### DISCUSSION TITLE
All cards in one column

### DISCUSSION THREAD

**Discussion by AseKarlsson** [mattieha/slider-button-card#18] [General]
Hi,
Just installed your Slider Button Card.
Like it very much, but all my cards is in the middle of the screen in 1 column.
I would like to get these in 3 columns instead of having to scroll down. Like standard Lovelace cards arrange, is this doable?
Also on iPad the slider graphics does not work correct (Its OK on PC and Android).
Also able to adjust the Color temperature from your cars would be nice, tried earlier to suggest something like this to original Lovelace cards: https://community.home-assistant.io/t/cusomizing-lovelace-light-card/291682
//Ase

**Comment by paulearley** [comment]
Place the slider-button-card in a grid card. then you can have 2 or three side-by-side in one column.
